utils/build/docker/dotnet/parametric/Endpoints/ApmTestApi.cs

```diff
@@ -35,7 +35,7 @@ public static void MapApmTraceEndpoints(WebApplication app, ILogger<ApmTestApi>
     private static readonly Type TracerType = Type.GetType("Datadog.Trace.Tracer, Datadog.Trace", throwOnError: true)!;
     private static readonly Type TracerManagerType = Type.GetType("Datadog.Trace.TracerManager, Datadog.Trace", throwOnError: true)!;
     private static readonly Type GlobalSettingsType = Type.GetType("Datadog.Trace.Configuration.GlobalSettings, Datadog.Trace", throwOnError: true)!;
-    private static readonly Type ImmutableTracerSettingsType = Type.GetType("Datadog.Trace.Configuration.ImmutableTracerSettings, Datadog.Trace", throwOnError: true)!;
+    private static readonly Type TracerSettingsType = Type.GetType("Datadog.Trace.Configuration.TracerSettings, Datadog.Trace", throwOnError: true)!;
```

Comment:
Should we do some sort of if/else based on the AssemblyVersion at runtime?

Reply:
Yup, thanks. I just hit this issue testing with latest release. Changing PR back to draft while I fix this.

Reply:
Done! I'll do more general reflection clean up in #3682.